src/test/java/podman/client/containers/ContainerCreateOptionsTest.java

```diff
@@ -53,4 +53,25 @@ void checkSomeEntries() {
         assertThat(json.getJsonArray("dns_server")).isNotNull().isNotEmpty().containsAll(List.of("1.1.1.1", "8.8.8.8"));
         assertThat(json.getBoolean("remove")).isTrue();
     }
+
+    @Test
+    void checkImageVolumes() {
+        JsonObject json = new ContainerCreateOptions()
+                .imageVolumes(List.of(
+                        new ContainerCreateOptions.ImageVolume("dest1", true, "source1", ""),
+                        new ContainerCreateOptions.ImageVolume("dest2", false, "source2", "sub")))
+                .json();
+
+        assertThat(json.getJsonArray("image_volumes"))
```

Comment:
We could also check that the subPath attribute is present, wdyt?

Reply:
This is what I have from https://docs.podman.io/en/latest/_static/api.html#tag/containers/operation/ContainerCreateLibpod:

![image](https://github.com/user-attachments/assets/74f0484b-23b9-4422-9e35-dd6c6f5b8133)

Reply:
Ah, ok, I see it. I see it works but given [we are using 5.0.0 version](https://github.com/jponge/podman-java-client/blob/ef35e81baa33263f82427d8f0b6d1cac035871d0/src/main/java/podman/client/PodmanClient.java#L16) let's see what @jponge thinks. 

Could you anyway add the corresponding assert to check this new field?

Reply:
Fixed